test/karma/test-app/components.d.ts

```diff
@@ -7,6 +7,8 @@
 import { HTMLStencilElement, JSXBase } from "@stencil/core/internal";
 import { SomeTypes } from "./util";
 import { TestEventDetail } from "./event-custom-type/cmp";
+export { SomeTypes } from "./util";
```

Comment:
Can you do me a favor and run `npm run build` from the `end-to-end` directory? I think some changes there haven't been generated

Reply:
https://github.com/ionic-team/stencil/pull/3625 created to catch these programmatically